### ISSUE TITLE
E2 Guardian - Auto switch to MITM to deliver block page

### ISSUE THREAD

**Issue by forid786** [marcelloc/Unofficial-pfSense-packages#52]
Hi,

The Group feature of Auto Switch to MITM to deliver block page doesn't seem to be working. For example, I have a client with the CA Cert installed, however their traffic is not being MITM'd, when they go to a blocked HTTPS site, I am only getting a browser connection closed error. No block page. I initially logged a issue on the E2 Guardian official repo, but later found out that this was a feature you added @marcelloc 

Can you take a look into this when possible please?

**Comment by forid786** [comment]
@Marcelloc have you got this working in any of your setups?

**Comment by marcelloc** [comment]
No, never tried. I just implemented it on gui when I saw the new feature on eguardian commits

**Comment by forid786** [comment]
E2 Guardian devs are saying it doesn't exist and isn't something they added. I'm now confused lol but it would be a nice feature to have either way if you could take a look.

**Comment by marcelloc** [comment]
I don't have time now to see when it was added to e2guardian code, but if it's not usable on current version, I'll remove from gui.

**Comment by forid786** [comment]
It doesn't seem to work at all. I'll have to create another issue on the official repo it seems. Maybe Fred didn't know about the feature.
